modules/dnn_modern/cmake/FindTBB.cmake

```diff
@@ -0,0 +1,283 @@
+# Locate Intel Threading Building Blocks include paths and libraries
```

Comment:
OpenCV has TBB support: https://github.com/opencv/opencv/blob/master/cmake/OpenCVDetectTBB.cmake
Are there any problems with reusing of mentioned code?